cognite/neat/_data_model/models/dms/_http.py

```diff
@@ -5,13 +5,23 @@
 
 from ._container import ContainerRequest
 from ._data_model import DataModelRequest
+from ._references import (
+    ContainerReference,
+    DataModelReference,
+    SpaceReference,
+    ViewReference,
+)
 from ._space import SpaceRequest
 from ._views import ViewRequest
 
 DataModelResource: TypeAlias = SpaceRequest | DataModelRequest | ViewRequest | ContainerRequest
 
 T_DataModelResource = TypeVar("T_DataModelResource", bound=DataModelResource)
 
+ResourceId: TypeAlias = SpaceReference | DataModelReference | ViewReference | ContainerReference
+
+T_ResourceId = TypeVar("T_ResourceId", bound=ResourceId)
```

Comment:
This is to match the DataModelResource to make it clear this is only space, data model, view and containers.